<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkUSNewVideoDeviceWidgetControls</class>
 <widget class="QWidget" name="QmitkUSNewVideoDeviceWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>837</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkUSNewVideoDeviceWidget</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="m_BoxMetadata">
     <property name="title">
      <string>Metadata:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Comment</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Device Information:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Manufacturer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_Manufacturer">
        <property name="text">
         <string>Unknown Manufacturer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="m_Model">
        <property name="text">
         <string>Unknown Model</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="m_Comment">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="m_GroupBoxVideoSource">
     <property name="title">
      <string>Video Source:</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="m_RadioDeviceSource">
        <property name="text">
         <string>From Device:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_DeviceSelector">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="m_RadioFileSource">
        <property name="text">
         <string>From File:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="m_FilePathSelector">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_OpenFileButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="m_RadioOIGTLServerSource">
        <property name="text">
         <string>OIGTL Server</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="m_OIGTLServerHost">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>localhost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_OIGTLServerPort">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>18944</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="m_RadioOIGTLClientSource">
        <property name="text">
         <string>OIGTL Client</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="m_OIGTLClientHost">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>localhost</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_OIGTLClientPort">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>18944</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Video Options:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="m_CheckGreyscale">
        <property name="text">
         <string>Greyscale Image (Significantly faster)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_BtnDone">
       <property name="text">
        <string>Add Video Device</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/USUI.qrc">
         <normaloff>:/USUI/accept.png</normaloff>:/USUI/accept.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_BtnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/USUI.qrc">
         <normaloff>:/USUI/restart.png</normaloff>:/USUI/restart.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>130</height>
      </size>
     </property>
     <property name="title">
      <string>Add Probes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QLineEdit" name="m_ProbesInformation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Enter the names of the probes of the VideoDevice and,
separated by commas, its scanning depths.
If you want to add several probes separate the probes
and their informations by semicolons
E.g. ACV,100,120,150;BDW,90,100;CSV,120,150,180
   </string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Override:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="m_ResolutionHeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>2048</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>480</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QCheckBox" name="m_CheckResolutionOverride">
        <property name="text">
         <string>Enable Resolution Override</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="m_ResolutionWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>2048</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>640</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>If you encounter problems with devices (e.g. Images of uniform color and error messages in the log window), then try to set the resolution externally using the device's driver panel and then enter the same resolution here.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QGroupBox" name="m_GroupBoxEditProbes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
     <property name="title">
      <string>Edit Probes</string>
     </property>
     <widget class="QComboBox" name="m_Probes">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="m_Depths">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_AddDepths">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Probes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scanning Depths for chosen Probe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Scanning Depths you want to add to chosen Probe.
Seperate them by commas. E.g. 120,150,180</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_BtnRemoveDepth">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove this Depth</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_BtnAddDepths">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Depths</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_BtnRemoveProbe">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove this Probe</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_Manufacturer</tabstop>
  <tabstop>m_Model</tabstop>
  <tabstop>m_Comment</tabstop>
  <tabstop>m_RadioDeviceSource</tabstop>
  <tabstop>m_DeviceSelector</tabstop>
  <tabstop>m_RadioFileSource</tabstop>
  <tabstop>m_CheckResolutionOverride</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/USUI.qrc"/>
 </resources>
 <connections/>
</ui>
